CppCoreGuidelines.md

```diff
@@ -3076,8 +3076,7 @@ The argument against is prevents (very frequent) use of move semantics.
 ##### Reason
 
 A return value is self-documenting as an "output-only" value.
-Note that C++ does have multiple return values, by convention of using a `tuple` (including `pair`),
-possibly with the extra convenience of `tie` at the call site.
+Note that C++ does have multiple return values, by convention of using a `tuple` (including `pair`), possibly with the extra convenience of `tie` or structured bindings (C++17) at the call site.
```

Comment:
both tie and structured bindings are mentioned lower in the Example section, but I suppose it doesn't hurt to have them directly in the "Reason"